bluebrain/repo-patches/packages/neuron/package.py

```diff
@@ -65,6 +66,7 @@ class Neuron(CMakePackage):
 
     # extra variants from coreneuron recipe
     variant('gpu', default=False, description="Enable GPU build")
+    variant('knl', default=False, description="Enable KNL specific flags")
```

Comment:
This should die, too. Do we actually use the KNLs for sims, still?

Reply:
Certainly! will follow that in the next PR!